hedera-node/src/main/java/com/hedera/services/txns/crypto/SideEffectsTracker.java

```diff
@@ -0,0 +1,10 @@
+package com.hedera.services.txns.crypto;
+
+import javax.inject.Inject;
+
+public class SideEffectsTracker {
```

Comment:
SideEffects tracker will most probably be non-injectable class, but rather we will create instances of it during TX execution